### ISSUE TITLE
Coordinates swapped on Kobo N905B after tapping in upper right

### ISSUE THREAD

**Issue by dygordon** [koreader/koreader#2448]
* KOReader version: 2015.11 stable
* Device: N905B

#### Issue
Screen coordinates are swapped, as described here: http://www.mobileread.com/forums/showpost.php?p=2674580&postcount=739
#### Steps to reproduce
Install KOReader. Start first time. Tap in upper right of screen. Coordinates are swapped.

**Comment by houqp** [comment]
Have you tried the latest nightly build? http://build.koreader.rocks/download/nightly/

**Comment by Frenzie** [comment]
Pinging @dygordon.

**Comment by dygordon** [comment]
Didn't get try the nightly build as I prefer stable builds!

Haven't updated since then if there is a new stable build out.

I forget exactly what I did to fix. There are other Kobo bug reports here
which mention swapping some numbers in a config or source file. Which is
what I did.


On Apr 2, 2017 9:40 AM, "Frans de Jonge" <notifications@github.com> wrote:

Pinging @dygordon <https://github.com/dygordon>.

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/koreader/koreader/issues/2448#issuecomment-290987089>,
or mute the thread
